pkg/cmd/tenant/variables/update/update.go

```diff
@@ -292,40 +293,40 @@ func PromptForVariable(opts *UpdateOptions, tenantVariables *variables.TenantVar
 	return selectedVariable, nil
 }
 
-func mapVariableControlTypeToVariableType(controlType variables.ControlType) (sharedVariable.VariableType, error) {
+func mapVariableControlTypeToVariableType(controlType resources.ControlType) (sharedVariable.VariableType, error) {
 	switch controlType {
-	case variables.ControlTypeSingleLineText, variables.ControlTypeMultiLineText:
+	case resources.ControlTypeSingleLineText, resources.ControlTypeMultiLineText:
```

Comment:
The variable.ControlType* usage required in deployments created a circular reference, this has been moved to resources.